src/components/Layout/Buttons/HelpButton.tsx

```diff
@@ -25,7 +26,7 @@ export const HelpButton: FunctionComponent<{ onPress: () => void }> = ({
       }}
     >
       <Feather name="compass" size={size(1.5)} color={color("blue", 40)} />
-      {" Need help?"}
+      {` ${i18n.t("loginScanCard.needHelp")}?`}
```

Comment:
On my phone it looks like the ? is cut off when it's chinese, is it the same on yours?

Reply:
Mine look's like this! Could it be because of iphone and android font differences😱?
![photo6309562019208079953](https://user-images.githubusercontent.com/38589870/94478575-090f7780-0206-11eb-91b5-6e7b0ed56dce.jpg)

Reply:
Is it cuz the ? is italics ah, mine is cut off halfway leh. "Need help" supposed to be italics right?

Reply:
Hmm the whole text is supposed to be in italics though:/ 
<img width="541" alt="Screenshot 2020-09-29 at 1 14 57 PM" src="https://user-images.githubusercontent.com/38589870/94515558-5e737500-0256-11eb-93dd-5646249bf672.png"> let me check why the chinese is not italics